hoki/utils/binning.py

```diff
@@ -0,0 +1,180 @@
+"""
+Routines to bin spectra.
+
+Author: Martin Glatzle
+"""
+import numpy as np
+from numba import jit
+
+
+def bin_spectra(wave, spectra, bins, edges=False):
```

Comment:
refactor `wave` to `wl` instead, it's just a convention I tend to use in all my codes and it's better to keep it consistent throughout hoki.

Reply:
Fixed by c96e92e6fa8f54883af7f07875494710c0c45dd9.